pkg/plugin/plugin_test.go

```diff
@@ -108,6 +109,9 @@ func TestPlatformPrepareCommand(t *testing.T) {
 		osStrCmp = "linux-ppc64le"
 	} else if os == "linux" && arch == "s390x" {
 		osStrCmp = "linux-s390x"
+	} 
+	else if os == "linux" && arch == "riscv64" {
```

Comment:
fix code style breaking CI

```suggestion
	} else if os == "linux" && arch == "riscv64" {
```